react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      "one": "hidden",
+      "two": "hidden",
+      "three": "hidden"
+    }
```

Comment:
While this is kind of dynamic it is still limiting to only 3 topics. We want to avoid limiting things, usually. In your sate you just need to track the `id` of the open item. For example:

```JAVASCRIPT
this.state = {
  openId: null
};
```